test/fixtures/code-exports/actual.js

```diff
@@ -1,8 +1,6 @@
-export default class Foo extends React.Component {}
-export default class extends React.Component {}
```

Comment:
What is this change related to?

Reply:
By the current spec only one default export can exist per module (I guess this has changed since this package was created). Babel now complains about this (maybe because the dependencies in this repo are not locked since it does not use yarn).